uefi/firmwarefile.go

```diff
@@ -98,6 +99,16 @@ type FirmwareFile struct {
 	ExtractPath string
 }
 
+// Assemble assembles the Firmware File from the binary
+func (f *FirmwareFile) Assemble() ([]byte, error) {
+	var err error
```

Comment:
I think lines 104-109 could simply be:

    return ioutil.ReadFile(f.ExtractPath)

Reply:
Never mind, I see that you're trying to store into `f.buf`.